source/features/prevent-link-loss.tsx

```diff
@@ -40,7 +49,7 @@ function getUI(field: HTMLTextAreaElement): HTMLElement {
 				{' by GitHub.'}
 			</>
 		),
-		classes: ['rgh-prevent-link-loss-container', 'flash-warn', 'my-2', 'mx-2'],
+		classes: ['rgh-prevent-link-loss-container', 'flash-warn', 'my-2', field.tagName === 'FORM' ? 'mx-2' : ''],
```

Comment:
```suggestion
		classes: [
			'rgh-prevent-link-loss-container',
			'flash-warn',
			'my-2',
			field.tagName === 'FORM' ? 'mx-2' : '',
		],
```

Reply:
Why doesn't eslint do that? 😿

Reply:
We don't run Prettier on JS code. I don't like how it affects all code even when it's not needed. _Maybe one day_